api/sixtyfps-cpp/include/sixtyfps.h

```diff
@@ -595,6 +616,13 @@ class VectorModel : public Model<ModelData>
         data.erase(data.begin() + index);
         this->row_removed(index, 1);
     }
+
+    /// Inserts the given value as a new row at the specified index
+    void insert_at(size_t index, const ModelData &value)
```

Comment:
Why is it called `insert_at`?  std::vector uses `insert`.
I guess that's because the vector::insert takes an iterator and not an index, but i'd still go with the standard terminology because:
 - we called the remove function `erase` , and not `erase_at`
 - one could argue that the iterator equivalent for the model is an int
 - string::insert is also taking a position.
 - that's what C++ developer are the most likely to look for.

Reply:
Okay. I don't have a strong feeling about it - reasoning as you mentioned also in the commit message. But I can change the commit to call it `insert()`.

Reply:
Done :)